data/json/items/tool/electronics.json

```
@@ -526,8 +524,7 @@
   {
     "id": "vibrator",
     "type": "TOOL",
-    "name": "vibrator",
-    "name_plural": "vibrators",
+    "name": { "str": "vibrator", "str_pl": "vibrators" },
```

```suggestion
    "name": { "str": "vibrator" },
```
Already follows normal pluralizing rules